routes/drones.js

```diff
@@ -1,36 +1,72 @@
-const express = require('express');
```

Comment:
In every route, remember to use the `next(error)` instead of `console.log` or `console.error` to send the error to the error handling middleware.